src/webview-bridge-cjs.js.flow

```diff
@@ -57,6 +57,19 @@ declare export function requestContact(
     },
 |}>;
 
+declare export function matchContactData(
+    numbers: Array<string>,
+): Promise<{|
+    matches: {
+        [key: string]: {
+            name?: string,
+            middleName?: string,
+            surname?: string,
+            encodedAvatar?: string,
+        },
+    },
```

Comment:
Use explicit exact or inexact object types, don't be implicit
```js
matches: {
    [key: string]: {|
        name?: string,
        middleName?: string,
        surname?: string,
        encodedAvatar?: string,
    |},
    ...
}
```

Reply:
ok!